<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorForm</class>
 <widget class="QWidget" name="ColorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: rgb(51, 51, 51);
  border-radius: 3px;
}

QToolTip {
  background-color: #2b2b2b;
  color: white;
  font: 9pt &quot;Microsoft YaHei UI&quot;;
}

QMenu {
  background-color: #2b2b2b;
  border-radius: 3px;
}

QMenu::item {
  border-radius: 3px;
  font-size: 12pt;
  color: white;
  padding: 2px 30px;
  margin: 5px 0px;
}

QMenu::item::selected {
  background-color: #606060;
}

#widget2, #widget3, #widgetCenter {
  border-radius: 0px;
}

#widget1 {
  border-top-right-radius: 0px;
  border-bottom-right-radius: 0px;
}

#widget4 {
  border-top-left-radius: 0px;
  border-bottom-left-radius: 0px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>66</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>66</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(58, 58, 58);
border-top-left-radius: 3px;
border-bottom-left-radius: 3px;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnPick">
        <property name="minimumSize">
         <size>
          <width>66</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>66</width>
          <height>34</height>
         </size>
        </property>
        <property name="toolTip">
         <string>拾取颜色</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: rgb(58, 58, 58);
  border-top-left-radius: 3px;
  background-position: center;
  background-repeat: no-repeat;
  background-image: url(:/icons/eye-dropper.png);
}
QPushButton:hover {
  background-color: rgb(78, 78, 78);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="minimumSize">
         <size>
          <width>66</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>66</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QListView {
  outline: 0px;
  background-color: rgb(58, 58, 58);
  border-bottom-left-radius: 3px;
}

QListView::item {
  background-color: #3a3a3a;
}

QListView::item:hover {
  background-color: #242424;
}

QListView::item:selected {
  background-color: #333333;
}
</string>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>45</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="widget1">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;选择颜色&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#widget1 {
  background-color: rgb(255, 255, 255);
}
#widget1:hover {
  background-color: rgb(207, 207, 207);
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="widget2">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;选择颜色&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#widget2 {
  background-color: rgb(219, 219, 219);
}

#widget2:hover {
 background-color: rgb(171, 171, 171);
 }</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="widgetCenter">
         <property name="minimumSize">
          <size>
           <width>144</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>144</width>
           <height>48</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;调整颜色&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#widgetCenter {
  background-color: rgb(181, 181, 181);
}
#widgetCenter:hover {
  background-color: rgb(133, 133, 133);
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="widget3">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;选择颜色&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#widget3 {
  background-color: rgb(130, 130, 130);
}
#widget3:hover {
  background-color: rgb(82, 82, 82);
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="widget4">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;选择颜色&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#widget4 {
  background-color: rgb(104, 104, 104);
}
#widget4:hover {
  background-color: rgb(70, 70, 70);
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QFrame" name="frameHex">
         <property name="minimumSize">
          <size>
           <width>336</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>336</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #454545;
border-radius: 2px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 700 9pt &quot;Microsoft YaHei UI&quot;;
color: rgb(121, 121, 121);</string>
            </property>
            <property name="text">
             <string>HEX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelHex">
            <property name="cursor">
             <cursorShape>IBeamCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Monaco&quot;;</string>
            </property>
            <property name="text">
             <string>b5b5b5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCopyHex">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="toolTip">
             <string>复制</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background-color: #454545;
  background-position: center;
  background-repeat: no-repeat;
  background-image: url(:/icons/clipboard-document-check.png);
}
QPushButton:hover {
  background-color: rgb(51, 51, 51);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameRgb">
         <property name="minimumSize">
          <size>
           <width>336</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>336</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #454545;
border-radius: 2px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 700 9pt &quot;Microsoft YaHei UI&quot;;
color: rgb(121, 121, 121);</string>
            </property>
            <property name="text">
             <string>RGB</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRgb">
            <property name="cursor">
             <cursorShape>IBeamCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Monaco&quot;;</string>
            </property>
            <property name="text">
             <string>rgb(181, 181, 181)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCopyRgb">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="toolTip">
             <string>复制</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background-color: #454545;
  background-position: center;
  background-repeat: no-repeat;
  background-image: url(:/icons/clipboard-document-check.png);
}
QPushButton:hover {
  background-color: rgb(51, 51, 51);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameHsl">
         <property name="minimumSize">
          <size>
           <width>336</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>336</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #454545;
border-radius: 2px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 700 9pt &quot;Microsoft YaHei UI&quot;;
color: rgb(121, 121, 121);</string>
            </property>
            <property name="text">
             <string>HSL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelHsl">
            <property name="cursor">
             <cursorShape>IBeamCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Monaco&quot;;</string>
            </property>
            <property name="text">
             <string>hsl(0, 0%, 71%)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCopyHsl">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="toolTip">
             <string>复制</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background-color: #454545;
  background-position: center;
  background-repeat: no-repeat;
  background-image: url(:/icons/clipboard-document-check.png);
}
QPushButton:hover {
  background-color: rgb(51, 51, 51);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>100</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
